cibyl/sources/zuul/source.py

```diff
@@ -171,3 +172,32 @@ def get_builds(self, **kwargs):
         :rtype: :class:`AttributeDictValue`
         """
         return self.get_jobs(**kwargs)
+
+
+class ZuulD(Source):
```

Comment:
The ZuulD source now appears duplicated, both here and in zuuld/source.py, but with differences, which is is the one that we want to keep?

Reply:
No doubt, it's duplicate. While reverting to fix _validate() i messed up in code.